azdev/utilities/path.py

```diff
@@ -200,6 +211,7 @@ def get_path_table(include_only=None, include_whl_extensions=False):
     table = {}
     cli_repo_path = get_cli_repo_path()
     ext_repo_paths = get_ext_repo_paths()
+    os.chdir(EXTENSIONS_DIR)
```

Comment:
The `EXTENSIONS_DIR` would be an empty string if setup wihtout -r. And, it will crash here

```Shell
$ cat ~/.azure/config
[cloud]
name = AzureCloud

[extension]
dev_sources = 

[core]
cache_ttl = 10
first_run = yes
collect_telemetry = no
```